content/articles/accessible-pdf-wheel.md

```diff
@@ -0,0 +1,229 @@
+* The twelve identity facets are arranged in a circle. While a sighted user
+would commonly read the wheel clockwise, the hierarchy or order of these facets
+is irrelevant.
+* The reflective prompts are also placed in the center of the circle for easy
+reference.
+
+{{< figure
+    src="/img/assets/wheel-first-structure.png"
+    caption="Initial visual layout for the Identity Wheel exercise"
+    alt="" >}}
+
+The experience through a screen reader, however, is linear, and the content of
+the Identity Wheel PDF needed to be in a logical, meaningful structure and
+sequential order regardless of its layout. I also had to take into account how
+the screen reader reads a PDF, and find solutions to its limitations.
+VoiceOver, for example, reads only the content that is visible on screen. If
+half of the wheel is visible on the monitor, then VoiceOver reads only half of
+the identity facets and reflective prompts.
+
+{{< figure
+    src="/img/assets/sr-visible-scope.jpg"
+    class="bordered"
+    alt="A diagram showing half of the PDF document on a monitor." >}}
+
+Determining the accessible experience for the Identity Wheel worksheet was an
+examination of the learning process of the exercise for _all learners_. What
+guidance and information did the learners require _beforehand_ to begin the
+exercise on their own? What did they need _while_ working on the activity?
+Would pagination be a problem if the worksheet is split into two pages? What
+would the linear layout of experience be like?
+
+We decided on the following experience structure and sequence:
+
+* The instructions and reflective prompts would be ahead of the Identity Wheel
+as guidance, on the first page.
+* The attribution would follow as reference, also on the first page.
+* The Identity Wheel would be on the second page for readability. The tabbing
+and reading order of the identity facets would be clockwise.
+* A duplicate of the reflective prompts remained in the center of the circle
+for sighted users, but would be read before the identity facets.
+
+{{< figure
+    src="/img/assets/wheel-second-structure.png"
+    caption="Revised visual layout for the Identity Wheel exercise.<br/>The numbers indicate order of reading by a screen reader as described above."
+    alt="" >}}
+
+## 1, 2, 3: Design, optimize, and test
+
+### 1. Design: Adobe InDesign
+
+PDF accessibility begins from the original source, and since the worksheet has
+a particular visual layout, I used Adobe InDesign to create the source
+document. I designed the worksheet according to the accessibility experience
+structure and order previously described. The key to the correct reading and
+tabbing order in Adobe InDesign is the Articles Panel, and Tag Panel for
+structure—headings, paragraphs, images. I made sure that the source document
+had no typos or errors, and that it complied to the accessibility experience
+_before exporting to interactive PDF_ to minimize repairs in subsequent steps.
```

Comment:
Add `an`: `before exporting to an interactive`.

Reply:
Thanks